hibike/hibike_process.py

```diff
@@ -6,15 +6,22 @@
 import glob
 import serial
 import os
+from platform import system
 
 __all__ = ["hibike_process"]
 
 
 uid_to_index = {}
 
 def hibike_process(badThingsQueue, stateQueue, pipeFromChild):
-
-    ports = glob.glob("/dev/ttyACM*") + glob.glob("/dev/ttyUSB*")
+    
+    # So that it's compatible with OS X Sierra
+    # Note: If you are running OS X Sierra, do not access the directory through vagrant ssh
+    # Instead access it through Volumes/vagrant/PieCentral
+    if system() == "Darwin":
```

Comment:
I don't remember what pre-Sierra macOS uses for USB device paths. Is it possible that it has either `ttyACM` or `ttyUSB` paths? Because in that case, it may make sense to just combine all the globs on one line instead of doing the `if` statement.